tests/integration/components/frost-bunsen-form/renderers/datetime-test.js

```diff
@@ -45,10 +46,15 @@ describe('Integration: Component / frost-bunsen-form / renderer / datetime', fun
   })
 
   it('renders as expected', function () {
-    expectCollapsibleHandles(0)
```

Comment:
mainly since we didn't see why it was being checked each time but we can add it back no problem